test/unit/Chainweb/Test/Pact5/RemotePactTest.hs

```diff
@@ -145,112 +162,134 @@ mkFixture v baseRdb = do
         , _serviceClientEnv = serviceClientEnv
         }
 
+class HasFixture where
+    remotePactTestFixture :: IO Fixture
+
+withFixture' :: IO Fixture -> ((CutFixture.HasFixture, HasFixture) => a) -> a
+withFixture' fixture tests =
+    withDict @HasFixture fixture $
+    CutFixture.withFixture' (_cutFixture <$> remotePactTestFixture) tests
+
+withSharedFixture :: ResourceT IO Fixture -> ((CutFixture.HasFixture, HasFixture) => TestTree) -> TestTree
+withSharedFixture mk tests =
+    withResourceT mk $ \fixture ->
+        withFixture' fixture tests
+
+withFixture :: Fixture -> ((CutFixture.HasFixture, HasFixture) => a) -> a
+withFixture fixture tests = withFixture' (return fixture) tests
```

Comment:
Should we just add this to `CutFixture.hs`?

Reply:
There is a variation on this for `CutFixture`, this is another kind of fixture so it gets its own flavor of `withFixture`.